0000-template-rfc.md

```diff
@@ -0,0 +1,57 @@
+- Feature Name: (fill me in with a unique ident, my_awesome_feature)
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- RFC PR: (leave this empty)
+- IPFS Issue: (leave this empty)
```

Comment:
From the [Couchbase SDK RFC template](https://github.com/couchbaselabs/sdk-rfcs/blob/7095a1e3278eaba6f89ab003b7dd918637bae3e5/0000-template.md) I like the additional fields:

 - Owner: [TBD: your name]
 - Current Status: TBD: DRAFT -> REVIEW -> ACCEPTED]

The status is described in the [Readme](https://github.com/couchbaselabs/sdk-rfcs/blob/7095a1e3278eaba6f89ab003b7dd918637bae3e5/README.md).

Reply:
Good idea. I added this, and rearranged the "lifecycle" section to spell out these stages.